spec/15-interfaces.md

```diff
@@ -230,14 +230,14 @@ This type is defined, as follows:
 
 ```PHP
 interface Throwable {
-  abstract public __toString(): string;
```

Comment:
I didn’t even notice while doing the string replace, this was already missing! Now it is fine. :smile: